negotiation/message/contract-negotiation-termination-message.json

```diff
@@ -2,7 +2,8 @@
   "@context":  "https://w3id.org/dspace/v0.8/context.json",
   "@type": "dspace:ContractNegotiationTerminationMessage",
   "dspace:processId": "urn:uuid:a343fcbf-99fc-4ce8-8e9b-148c97605aab",
-  "dspace:code": "...",
+  "dspace:code": {
```

Comment:
Why is this a JSON Object now? I thought it should be a free-text field (=string)...

Reply:
reverted back